public/docs/ts/latest/glossary.jade

```diff
@@ -343,9 +333,9 @@ block includes
 
     The latest approved version of JavaScript is
     [ECMAScript 2016](http://www.ecma-international.org/ecma-262/7.0/)
-    (AKA "ES2016" or "ES7") and many Angular 2 developers will write their applications
+    (AKA "ES2016" or "ES7") and many Angular 2 developers write their applications
     either in this version of the language or a dialect that strives to be
-    compatible with it such as [TypeScript](#typesScript).
+    compatible with it, such as [TypeScript](#typesScript).
```

Comment:
Fixed. Searched throughout angular.io for any other occurrences, but there were no others (whew!).